files/en-us/web/api/element/removeattribute/index.html

```diff
@@ -62,8 +62,7 @@ <h2 id="Specifications">Specifications</h2>
   </thead>
   <tbody>
     <tr>
-      <td>{{SpecName('DOM WHATWG', '#dom-element-removeattribute', 'Element"
-        removeAttribute')}}</td>
+      <td>{{SpecName('DOM WHATWG', '#dom-element-removeattribute', 'Element" removeAttribute')}}</td>
```

Comment:
```suggestion
      <td>{{SpecName('DOM WHATWG', '#dom-element-removeattribute', 'Element: removeAttribute')}}</td>
```
typo that we may as well fix while we’re in here…